dask_match/tests/test_collection.py

```diff
@@ -139,12 +139,97 @@ def test_repr(ddf):
 
 
 def test_columns_traverse_filters(df, ddf):
-    result = optimize(ddf[ddf.x > 5].y)
+    result = optimize(ddf[ddf.x > 5].y, fuse=False)
     expected = ddf.y[ddf.x > 5]
 
     assert str(result) == str(expected)
 
 
+def test_optimize_fusion(ddf):
+    ddf2 = (ddf["x"] + ddf["y"]) - 1
+    unfused = optimize(ddf2, fuse=False)
+    fused = optimize(ddf2, fuse=True)
+
+    # Should only get one task per partition
+    assert len(fused.dask) == ddf.npartitions
+    assert_eq(fused, unfused)
```

Comment:
calling `assert_eq` will call optimize with the default `fuse=True`, correct?  If so, are these `assert_eq(fused, unfused)` doing what we expect?

Reply:
I don’t think we configured compute to call our optimize function yet (I think that’s what you’re asking)

Reply:
Ah, good point.  Thanks.